src/feature_flags.js

```diff
@@ -0,0 +1,13 @@
+// List of supported flags and their default value.
+const FLAGS = {}
+
+const getFlags = (input = {}, flags = FLAGS) =>
```

Comment:
[sand] Would `(input, flags = FLAGS) => ({ ...flags, ...input })` (or `(input, flags = FLAGS) => ({ ...flags, ...removeFalsy(input) })` if `input` might contain `undefined` keys) be simpler? 
One difference is that if `input` contains unknown flags, those would not be filtered, but that should not be a problem?

Reply:
I would actually like `FLAGS` to contain all the feature flags that ZISI is currently accepting, and make it so that any supplied flags that don't belong to that list are ignored (e.g. https://github.com/netlify/zip-it-and-ship-it/pull/567/files#diff-84cbdd132bc7ff6db27e194fa841b40864de4a8f661e0a79273ff9b2477e9e3dR11-R15). I think this is important to ensure that any flags used throughout the codebase are declared in one place for greater visibility.

Reply:
I'll merge this as is for now, but I'm happy to rework this later if you think we should.

Reply:
Sounds good :+1: